CMakeLists.txt

```
@@ -441,13 +472,28 @@ if(BUILD_TEST)
   set(NVFUSER_TESTS_KERNELS "${NVFUSER_TESTS}_kernels")
 
   add_library(${NVFUSER_TESTS_KERNELS} SHARED ${NVFUSER_ROOT}/test/rng_kernels.cu)
-  # CUDA 11 does not support C++20, so hard code C++17 here
-  set_property(TARGET ${NVFUSER_TESTS_KERNELS} PROPERTY CXX_STANDARD 17)
+  set_target_properties(${NVFUSER} PROPERTIES
+    C_STANDARD 17
```

Should we use the following?
```suggestion
    C_STANDARD ${NVFUSER_CPP_STANDARD}
```

---

C standards and C++ standards are not necessarily equivalent (they happen to have coincided in years, lately, but there's no guarantee there).